src/dropdown.ts

```diff
@@ -777,7 +783,7 @@ export default class GlideCoreDropdown
                   aria-controls="options"
                   aria-describedby="description"
                   aria-expanded=${this.open && !this.disabled}
-                  aria-labelledby="selected-option-labels label ${this
+                  aria-labelledby="selected-option-labels label loading-feedback ${this
```

Comment:
I thought it would be good to move away from "loading" because it reads more like a state and because it's tightly coupled to the type of animation shown.